Sources/Modules/CagedDomain/Everglow.CagedDomain.csproj

```diff
@@ -1,5 +1,10 @@
 
     <Project Sdk="Microsoft.NET.Sdk">
       <Import Project="..\Module.props" />
+	<ItemGroup>
+	  <Reference Include="SubworldLibrary">
+	    <HintPath>..\..\..\Libraries\SubworldLibrary.dll</HintPath>
+	  </Reference>
+	</ItemGroup>
```

Comment:
why does CagedDomain reference Subworld Library? Are you planning on adding subworld functions to that module?

Reply:
yes，this branch maybe work with a subWorld.